hudi-common/src/main/java/org/apache/hudi/common/util/FailSafeConsistencyGuard.java

```diff
@@ -95,7 +95,7 @@ public void waitForFilesVisibility(String dirPath, List<String> files, FileVisib
 
         switch (event) {
           case DISAPPEAR:
-            LOG.info("Following files are visible" + candidateFiles);
+            LOG.info("Following files are visible{}", candidateFiles);
```

Comment:
add a blank would be better `LOG.info("Following files are visible {}", candidateFiles);`

Reply:
Ok, change it now.